pkg/scanners/scannerv4/convert.go

```diff
@@ -134,8 +132,6 @@ func setCvss(vuln *storage.EmbeddedVulnerability, cvss *v4.VulnerabilityReport_V
 	}
 	if v3 := cvss.GetV3(); v3 != nil {
 		if c, err := cvssv3.ParseCVSSV3(v3.GetVector()); err == nil {
-			c.ExploitabilityScore = 0.0 // TODO cvssv3.ExploitabilityScore(v3.GetVector())
-			c.ImpactScore = 0.0         // TODO cvssv3.ImpactScore(v3.GetVector())
 			c.Score = v3.GetBaseScore()
```

Comment:
do we still need this call? (similarly, do we want to verify our results match?)